packages/xo-web/src/xo-app/disk-import/index.js

```diff
@@ -108,12 +109,16 @@ const DiskImport = decorate([
         <Row>
           <LabelCol>{_('importToSr')}</LabelCol>
           <InputCol>
-            <SelectSr onChange={effects.onChangeSr} required value={sr} />
+            <SelectSr onChange={effects.onChangeSr} required value={sr} allowIsoSr />
           </InputCol>
         </Row>
         {sr !== undefined && (
           <div>
-            <Dropzone onDrop={effects.handleDrop} message={_('dropDisksFiles')} />
+            <Dropzone
+              onDrop={effects.handleDrop}
+              message={_('dropDisksFiles')}
+              accept={sr.content_type === 'iso' ? '.iso' : '.vhd,.vmdk'}
+            />
```

Comment:
Is that intentional to use a string instead an array ?
```suggestion
            <Dropzone
              onDrop={effects.handleDrop}
              message={_('dropDisksFiles')}
              accept={sr.content_type === 'iso' ? '.iso' : ['.vhd', '.vmdk']}
            />
```